Directory.Build.props

```diff
@@ -3,7 +3,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta-63127-02" PrivateAssets="All" />
     <PackageReference Include="OpenCover" Version="4.6.519" PrivateAssets="All" />
-    <PackageReference Include="ReportGenerator" Version="3.1.2" PrivateAssets="All" />
+    <PackageReference Include="ReportGenerator" Version="4.0.2" PrivateAssets="All" />
```

Comment:
This is a breaking update and needs changes to `Build.ps1` to update this code: https://github.com/justeat/JustSaying/blob/63043bca00bb3c18c3119d20ba169442ac37f0be/Build.ps1#L112-L133